linera-service/src/cli/main.rs

```diff
@@ -684,10 +696,87 @@ impl Runnable for Job {
                                 }
                                 RemoveValidator { public_key } => {
                                     if validators.remove(&public_key).is_none() {
-                                        warn!("Skipping removal of nonexistent validator");
+                                        warn!("Validator {public_key} does not exist; aborting.");
                                         return Ok(ClientOutcome::Committed(None));
                                     }
                                 }
+                                ChangeValidators {
+                                    add_validators,
+                                    modify_validators,
+                                    remove_validators,
+                                    skip_online_check: _,
+                                } => {
+                                    // Validate that all validators to add do not already exist.
+                                    for validator in &add_validators {
+                                        if validators.contains_key(&validator.public_key) {
+                                            warn!(
+                                                "Cannot add existing validator: {}. Aborting operation.",
```

Comment:
same as above, if we abort the command, this should be an error, otherwise just leave the warning without the aborting part.